recipes/fiji-ilastik/set_ilastik_executable_in_ImageJ.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Update the executable path:
+# Version 1.4.0b29
+ilastik_bin=${CONDA_PREFIX}/bin/ilastik
+if [ ! -e "$ilastik_bin" ]; then
+  # Version 1.4.0b27
+  ilastik_bin=${CONDA_PREFIX}/bin/ilastik-app
+  if [ ! -e "$ilastik_bin" ]; then
+    ilastik_bin=""
+  fi
+fi
```

Comment:
see comment above, I'd rather not introduce this and support only ilastik 1.4.0b29 onwards...

Reply:
same